plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -12,4 +13,14 @@
          requiredSourceLevel="1.5">
      </compilationParticipant>
   </extension>
+  
+  <extension 
+      id="blacklistedClass" 
+      name="App Engine Whitelist Marker"
+      point="org.eclipse.core.resources.problemmarkers">
```

Comment:
You could embed the bundle symbolic name directly.  By referencing a class in that bundle, you insulate the test in the face of bundle renames.